### ISSUE TITLE
[1.17] Allow Config for Extending Effect Duration

### ISSUE THREAD

**Issue by SamuelDancing** [kwpugh/Ward-Blocks#2]
So I grabbed this mod because I'm playing with Natural Regeneration disabled. The problem is that some effects like regeneration need at least 2.5 seconds (125 ticks) in order to have any actual benefit. The result is that the Heal Ward doesn't actually restore health, just hunger. Also, the part where the blocks are redstone operated is a little too close to the "Where to Find Them" section which makes it really easy to miss.

TLDR: Regeneration I needs to last 125 ticks to be of any worth.

**Comment by kwpugh** [comment]
I don't understand what you mean.  I have played with it countless times and it restores read hearts and hunger legs.  Put a Redstone lever on it, flip the switch and go.   Are you using a button or something?

**Comment by SamuelDancing** [comment]
If you're still confused, what I mean is: The Healing Ward applies effects like it should, but the regeneration doesn't last for a half-heart heal.

To see what I mean, use `/gamerule naturalRegeneration False`, reduce your health below full, then enable the ward and monitor your health for 10+ seconds.

**Comment by kwpugh** [comment]
Ok, I see what you mean.   I'll take a look but I don't think I want to bump up its timing since it is running in a blockeuntity tick method.   I'll see what I can work with.

**Comment by kwpugh** [comment]
ok, I'm playing around with a config option to change the effect duration, however, because this is all running within a block entity tick() method, if you game is running at 20TPS, the regeneration will be very, very fast.

I don't want to get into adding an artificial delay into the tick(), so there it is.

**Comment by kwpugh** [comment]
Added some options to latest release.

**Comment by SamuelDancing** [comment]
Thank you! I'll close this issue for now seeing as how it has been solved.
